Automattic/jetpack
# Migrate Jetpack multisite callables from Sync package to Jetpack Plugin proper

In #18797 we didn't migrate the multisite callables from the Sync package. This PR completes the migration so that Sync does not reference functionality not contained in its package or dependencies.

#### Changes proposed in this Pull Request:
* No functional changes, simply shifting addition of Jetpack multisite callables into Jetpack.

Does this pull request change what data or activity we track or use?
No
Testing instructions:
These changes are covered under the existing test test_sync_callable_whitelist

Manual Test
* yarn docker:wp shell --url=DOMAIN
* \Automattic\Jetpack\Sync\Defaults::$default_multisite_callable_whitelist; 
* Verify empty array returned
* \Automattic\Jetpack\Sync\Defaults::get_multisite_callable_whitelist();
* Verify network_name, network_allow_new_registrations, network_add_new_users, network_site_upload_space, network_upload_file_types and network_enable_administration_menus are returned. 

#### Proposed changelog entry for your changes:
* N/A - no functional changes simply shifting code :)
